From 544abc16419065c009361b9627f69665471cfb6d Mon Sep 17 00:00:00 2001
From: Nathan Owens <ndowens@artixlinux.org>
Date: Fri, 20 Nov 2020 22:24:08 -0600
Subject: [PATCH] Disable try_run

If cross-building, try_run cannot run during
cross-build so cause build to fail
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git CMakeLists.txt CMakeLists.txt
index 2d860d4..c14a203 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -61,7 +61,7 @@ if("${CMake_SOURCE_DIR}" STREQUAL "${CMAKE_SOURCE_DIR}")
     if(CMAKE_TRY_COMPILE_OSX_ARCHITECTURES)
       message(FATAL_ERROR "TRY_RUN not allowed with CMAKE_TRY_COMPILE_OSX_ARCHITECTURES=[${CMAKE_TRY_COMPILE_OSX_ARCHITECTURES}]")
     else()
-      _TRY_RUN(${ARGV})
+			#  _TRY_RUN(${ARGV})
     endif()
   endmacro()
 endif()
-- 
2.29.2

